### DISCUSSION TITLE
How can i configure a real-time library to run only in background, without any user request?

### DISCUSSION THREAD

**Discussion by wemersonrv** [adonisjs/core#1303] [Help] [answered]
Hello guys,

How can i configure a real-time library to run only in background, without any user request?

Example: 

I need to use this lib: https://github.com/pedroslopez/whatsapp-web.js with AdonisJS 4.1

If i create an empty nodejs project without framework, it's fine 'cause does not have a lot of MVC stuffs on it... but with AdonisJS, there are some settings rules i need to follow and i don't know how to do that.

**Comment by McSneaky** [comment, marked as answer]
Not really sure, how this library works, but it looks like you could start it up at provider boot time
https://adonisjs.com/docs/4.1/service-providers

**Reply by wemersonrv** [reply]
> Not really sure, how this library works, but it looks like you could start it up at provider boot time
> https://adonisjs.com/docs/4.1/service-providers

Tx... it works.
